core/utils.py

```diff
@@ -109,7 +110,7 @@ def resize_image(im, edge, format):
     (w, h) = im.size
     (width, height) = scale_dimension(w, h, long_edge=edge)
     content = BytesIO()
-    im = im.resize((width, height), PIL.Image.ANTIALIAS)
+    im = im.resize((width, height), Resampling.LANCZOS)
```

Comment:
C'est possible d'avoir un commentaire de c'est qui LANCZOS ?
Auntialias j'ai une bonne idée de ce que ça fait mais là c'est un peu plus compliqué

Reply:
Le message de déprécation de PIL disait qu'ANTIALIAS était déprécié et qu'il fallait utiliser LANCZOS à la place. Donc j'ai fait en vérifiant que ça marchait,

Concrètement, à la base, LANCZOS était l'algorithme d'upscale par défaut. Depuis, les devs de Pillow en ont rajouté d'autres et ont déprécié ANTIALIAS. https://pillow.readthedocs.io/en/stable/releasenotes/2.7.0.html#antialias-renamed-to-lanczos

Et sinon, ce que c'est le filtre lanczos : j'ai pas compris. C'est de la mathémagie. Le TC, c'est trop loin pour moi, les maths aussi.

Reply:
`im = im.resize((width, height), Resampling.LANCZOS) # Resize with anti-aliasing`
Je pense que ça suffit et ce sera plus clair